assets/lib/share-link/share-link.js

```diff
@@ -1,25 +1,23 @@
 /**
  * By Elementor Team
  */
-( function( $ ) {
-	window.ShareLink = function( element, userSettings ) {
-		var $element,
-			settings = {};
+( function() {
```

Comment:
Could we remove this function wrapper?
In that case should we create it as a class? 

Not sure. But this looks like a jQuery approach.

Reply:
This approach called Immediately Invoked Function Expression (IIFE), it's an older convention. I agree with @hein-obox , we need to stop using IIFE.

Ref:
* https://developer.mozilla.org/en-US/docs/Glossary/IIFE
* https://en.wikipedia.org/wiki/Immediately_invoked_function_expression
* Also Google for "IIFE alternatives".